File: vmworld2020/go-outline/handler/handler.go
package function

import (
	"net/http"

	handler "github.com/openfaas/templates-sdk/go-http"
)

// Handle a function invocation
func Handle(req handler.Request) (handler.Response, error) {
	// Parse the event

	// Read the config

	// Implement business logic

	// Handle function response
	return handler.Response{
		Body:       req.Body,
		StatusCode: http.StatusOK,
	}, nil
}

File: vmworld2020/py-outline/handler/handler.py

def handle(req):
    """handle a request to the function
    Args:
        req (str): Cloud Event as String
    """

    print(f'Event (raw) > {req}')
    #parse event
    

    PAGERDUTY_CONFIG = '/var/openfaas/secrets/pdconfig'
    #read the config
    

    #implement business logic
    PAGERDUTY_API = 'https://events.pagerduty.com/v2/enqueue'
    #make a POST Rest API request to pagerduty
    


    return
